lib/rules/jsdoc.js

```diff
@@ -22,8 +22,8 @@
  *
  * There are 3 presets for `Closure Compiler`, `JSDoc3` and `JSDuck5`.
  *
- * By default it allows any tag of mixed set. You can pass `Object`
- * to select preset with `preset` field and add custom tags with `extra` field.
+ * By default it allows any tag of mixed set. You can pass `Object` to select
+ * preset with `preset` field and add custom tags with `extra` field.
```

Comment:
> Is there another word for tag

According to http://usejsdoc.org/about-block-inline-tags.html tag is correct.

> Maybe `By default, it allows for any combination of tags?`

Does this meant that the user can use any tag of any existing preset? Atm preset field works like clarification. And by default it allows any tag of combined/merged tag set. Not sure that your version is more correct. But feels like the current one is even worse.

There is one more problem with this file because it's generated by the script at the end. If we want to make here changes we should do it also in jsdoc repo. Just fyi. ;-) Guess would be better to check the whole docs for grammar.